### DISCUSSION TITLE
Driver installation

### DISCUSSION THREAD

**Discussion by MatteoHatteo** [seerge/g-helper#1917] [Q&A] [answered]
Hello! Under the updates tab, I see that my Dolby Atmos and Fortemedia drivers are not updated yet. I clicked to get the .zip files but running the .bat files or .exe files within doesn't show any clear prompt that the drivers are installed, and I can't find them in device manager. Any idea how you install these or are they already installed?
![image](https://github.com/seerge/g-helper/assets/156571462/4f331363-cb6a-486e-b6ba-3a6397fd48ae)

Thanks!

**Comment by seerge** [comment, marked as answer]
@MatteoHatteo Hello :) that's just a mirror of Asus website. Most probably you need to launch .bat file (i remember doing something similar for my audio drivers). To be certain - launch bat as admin.

**Reply by MatteoHatteo** [reply]
Seems like launching the .bat files as admin did the job :D Thank you so much for this excellent tool!
